requirements.txt

```diff
@@ -100,10 +102,10 @@ urllib3==1.26.8; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.
 uvicorn[standard]==0.17.6
 uvloop==0.16.0
 watchdog==2.1.6
-watchgod==0.7
+watchgod==0.8
 wcwidth==0.2.5
 websockets==10.2
 whitenoise==6.0.0
 whoosh==2.7.4
-zipp==3.7.0; python_version < '3.10'
+zipp==3.7.0; python_version < '3.9'
```

Comment:
What do you think about this @stumpylog?

Reply:
That looks like it's just matching to the marker I added in #348, so it makes sense to me.  Once the tests pass, of course, especially for 3.8

Reply:
It actually came into dev with #332, which did pass tests.